src/backend/TrafficCourts/Workflow.Service/Consumers/DisputeApprovedConsumer.cs

```diff
@@ -0,0 +1,97 @@
+﻿using MassTransit;
+using TrafficCourts.Messaging.MessageContracts;
+using TrafficCourts.Workflow.Service.Services;
+using TrafficCourts.Workflow.Service.Models;
+
+namespace TrafficCourts.Workflow.Service.Consumers
+{
+    /// <summary>
+    ///     Consumer for DisputeApproved message.
+    /// </summary>
+    public class DisputeApprovedConsumer : IConsumer<DisputeApproved>
+    {
+        private readonly ILogger<DisputeApprovedConsumer> _logger;
+        private readonly ISubmitDisputeToArcService _submitDisputeToArcService;
+
+        public DisputeApprovedConsumer(ILogger<DisputeApprovedConsumer> logger, ISubmitDisputeToArcService submitDisputeToArcService)
+        {
+            _logger = logger;
+            _submitDisputeToArcService = submitDisputeToArcService;
+        }
+        public async Task Consume(ConsumeContext<DisputeApproved> context)
+        {
+            try
+            {
+                if (context.RequestId != null)
+                {
+                    _logger.LogInformation("DisputeApprovedConsumer is consuming message: " + context.Message.Id);
```

Comment:
Maybe Phil can comment, but VS Code is recommending this instead:
```suggestion
                    _logger.LogInformation($"DisputeApprovedConsumer is consuming message: {context.Message.Id}");
```

Reply:
Sorry that's a bit of old Java habit for me. However, as far as I know since Phil mentioned about it before, using string interpolation is considered bad form for Serilog, because it produces an unstructured log message with  'context.Message.Id' formatted into the string before Serilog sees it. Therefore, a better alternative would be structured logging such as: _logger.LogInformation("DisputeApprovedConsumer is consuming message: {MessageId} ", context.Message.Id);

I will make that change and commit.